cpu/efm32/periph/timer.c

```diff
@@ -106,12 +120,12 @@ int timer_set_absolute(tim_t dev, int channel, unsigned int value)
         return -1;
     }
 
-    if (value > 0xffff) {
+    if (value > TIMER_TopGet(timer_config[dev].timer.dev)) {
```

Comment:
Why not `0xFFFFFFFF` like before?

Reply:
Thanks for the explanation @chrysn, I'd prefer then the change as is it right now.